tests/providers/google/cloud/sensors/test_gcs.py

```diff
@@ -72,7 +72,7 @@ def context():
     """
     Creates an empty context.
     """
-    context = {"data_interval_end": datetime.utcnow()}
+    context = {"data_interval_end": datetime.now(tz=timezone.utc)}
```

Comment:
I guess it’s fine since the tests never complained before this, but `data_interval_end` is documented (and is in reality) an aware Pendulum DateTime in UTC.

Reply:
You suggest to use `pendulum.DateTime` here? That should not be a problem, we could use `airflow.utils.timezone.utcnow` here